scapy/layers/ntp.py

```diff
@@ -394,6 +394,9 @@ def getfield(self, pkt, s):
             extensions_len = len(extensions)
             while extensions_len >= 16:
                 ext_len = struct.unpack("!H", extensions[2:4])[0]
+                ext_len = min(ext_len, extensions_len)
+                if ext_len < 1:
+                    ext_len = extensions_len
```

Comment:
Sorry, this is a mistake. These tests come from PR #247 . This has been fixed in the latest commit.